.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
+dist: xenial
```

Comment:
Not sure of impact in travis, but should we be using latest i.e cosmic?
http://releases.ubuntu.com/

Reply:
Thanks for reviewing, @bwalsh!